docs/evaluation/how_to_guides/evaluation/evaluate_with_attachments.mdx

```diff
@@ -0,0 +1,103 @@
+import {
+  CodeTabs,
+  python,
+  typescript,
+  PythonBlock,
+  TypeScriptBlock,
+} from "@site/src/components/InstructionsWithCode";
+
+# Evaluate an LLM application with attachments
+
+Attachments allow you to associate large files with your examples. This allows you to evaluate RAG applications
```

Comment:
maybe worth adding a sentence about why attachments are better than storing file in example inputs